clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -976,6 +976,28 @@ static QualType GetSYCLKernelObjectType(const FunctionDecl *KernelCaller) {
   return KernelParamTy.getUnqualifiedType();
 }
 
+static CXXMethodDecl *getOperatorParens(const CXXRecordDecl *Rec) {
+  for (auto *MD : Rec->methods()) {
+    if (MD->getOverloadedOperator() == OO_Call)
+      return MD;
+  }
+  return nullptr;
+}
+
+// Fetch the associated call operator of the kernel object
+// (of either the lambda or the function object).
+CXXMethodDecl *
```

Comment:
Really sorry for not catching this earlier.
```suggestion
static CXXMethodDecl *
```

Reply:
No worries. Thanks for the reviews!